SU-SWS/saa_victoria_subtheme
# ADAPT-476: Added Victoria screenshot for the theme.

# READY FOR REVIEW

# Summary
- Updated screenshot to show 'Victoria'

# Review By (Date)
- 7/17

# Urgency
- wanted before 7/21

# Steps to Test

1. check out branch
1. clear caches and maybe clear chrome cache (i needed to)
2. maybe turn theme on and off to refresh if needed
3. see victoria screenshot on appearances page

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket 476 testing small tweaks
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
